framework/web/CHttpRequest.php

```diff
@@ -287,7 +293,9 @@ public function getRestParams()
 		if($this->_restParams===null)
 		{
 			$result=array();
-			if(function_exists('mb_parse_str'))
+			if ($this->getContentType() === 'application/json')
```

Comment:
this should check only the beginning of content type to also support json which has the addition `; charset=utf8` like this: `strncmp($this->getContentType(), 'application/json', 16) === 0`